apps/condo/domains/ticket/utils/handlers.js

```diff
@@ -27,24 +44,25 @@ const detectEventTypes = ({ operation, existingItem, updatedItem }) => {
     const nextAssigneeId = get(updatedItem, 'assignee')
     const nextExecutorId = get(updatedItem, 'executor')
     const nextStatusId = get(updatedItem, 'status')
+    const client = get(updatedItem, 'client')
     const result = {}
 
     /**
      * assignee connected within create ticket operation or
      * assignee connected/changed within update ticket operation
      */
-    result[ASSIGNEE_CONNECTED_EVENT_TYPE] = isCreateOperation && !!nextAssigneeId || isUpdateOperation && !!nextAssigneeId && nextAssigneeId !== prevAssigneeId
+    result[ASSIGNEE_CONNECTED_EVENT_TYPE] = !!nextAssigneeId && (isCreateOperation || isUpdateOperation && nextAssigneeId !== prevAssigneeId)
 
     /**
      * executor connected within create ticket operation or
      * executor connected/changed within update ticket operation
      */
-    result[EXECUTOR_CONNECTED_EVENT_TYPE] = isCreateOperation && !!nextExecutorId || isUpdateOperation && nextExecutorId && nextExecutorId !== prevExecutorId
+    result[EXECUTOR_CONNECTED_EVENT_TYPE] = !!nextExecutorId && (isCreateOperation || isUpdateOperation && nextExecutorId !== prevExecutorId)
 
     /**
      * ticket status changed
```

Comment:
please add more detailed comment, fully describing business logic, because it's not obvious